receiver/windowsperfcountersreceiver/config.go

```diff
@@ -46,23 +72,73 @@ func (c *Config) Validate() error {
 		errs = multierr.Append(errs, fmt.Errorf("must specify at least one perf counter"))
 	}
 
+	if len(c.MetricMetaData) == 0 {
+		errs = multierr.Append(errs, fmt.Errorf("must specify at least one metric"))
+	}
+
+	for _, metric := range c.MetricMetaData {
+		if metric.MetricName == "" {
+			errs = multierr.Append(errs, fmt.Errorf("a metric does not include a name"))
+			continue
+		}
+		if metric.Description == "" {
+			errs = multierr.Append(errs, fmt.Errorf("metric %q does not include a description", metric.MetricName))
+		}
+		if metric.Unit == "" {
+			errs = multierr.Append(errs, fmt.Errorf("metric %q does not include a unit", metric.MetricName))
+		}
```

Comment:
Can any of these be omitted? Seems unnecessary to require everything. In the context of `mdatagen` this kind of thing is reasonable because we are requiring developers to fully specify the scrapers they are writing. However, in this receiver we would be placing requirements on a user. I think we should err on the side of supporting sane defaults here.

Reply:
There's nothing pulled from the scrapers for description or unit, should those just be defaulted to an empty description and maybe `1` for the unit? should we have a default unit type if both are empty? maybe a double gauge because that's what everything was set as previously?

Reply:
The description and unit defaults you proposed sound right to me. 

Defaulting the data type to gauge seems reasonable too, since gauges have fewer constraints and details. As much as we want to encourage precise data modeling, it's a high bar to put on users.

Reply:
Added